java/examples/erparticle/src/main/java/com/nedap/retail/services/examples/WorkfolwExample.java

```diff
@@ -0,0 +1,64 @@
+package com.nedap.retail.services.examples;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.joda.time.DateTime;
+
+import com.nedap.retail.messages.Client;
+import com.nedap.retail.messages.workflow.WorkflowEvent;
+import com.sun.jersey.api.client.UniformInterfaceException;
+
+public class WorkfolwExample {
+
+    public static void runExample(final Client client) {
+        System.out.println("-------------");
+        System.out.println("Workflow API example");
+        System.out.println("-------------");
+
+        final String location = "http://location-testing";
```

Comment:
That is not useful here as this is example code. An actual implementation would use multiple locations and get this from the database.